django/contrib/gis/geos/factory.py

```diff
@@ -15,12 +17,10 @@ def fromfile(file_h):
 
     # If we get WKB need to wrap in memoryview(), so run through regexes.
     if isinstance(buf, bytes):
-        try:
+        with suppress(UnicodeDecodeError):
             decoded = buf.decode()
```

Comment:
I've reverted this change, and `srs.py` as well. We can close the ticket as wontfix, possible. The effort was not completely in vain since it spawned a new ticket about the Oracle datatypes.